src/Umbraco.Cms.Integrations.Crm.ActiveCampaign/Models/Dtos/ResponseDto.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Newtonsoft.Json;
+
+using System.Text.Json.Serialization;
+
+namespace Umbraco.Cms.Integrations.Crm.ActiveCampaign.Models.Dtos
+{
+    public class ResponseDto<T>
+        where T : class
+    {
+        [JsonProperty("data")]
```

Comment:
Also here the point about using `Newtonsoft.Json` or `System.Text.Json` throughout, but not both.